### PR TITLE
🐛  Resolve Windows Docker Bug & 🐍  Upgrade to Python 3.9

### LINKED ISSUE #180: 🐞 Windows Support for Docker Make
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When you run `make docker-run` or `make docker-dev` per the readme on Windows you get the error `COMPOSE_DOCKER_CLI_BUILD' is not recognized as an internal or external command, operable program or batch file.`

### Expected Behavior

It should run

### Steps To Reproduce

1. run `make docker-run` or `make docker-dev` in Windows

### Environment

```markdown
- OS: Windows
```

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by lprichar**
### Issue
Fixes #180 

### Description
- Removes unnecessary variables that won't run on Windows
- Clarifies ways to run the app in the readme
- Exposes 8081 as 8181 to avoid conflicts on Windows 11

### Testing
Run `make docker-run` or `make docker-dev` on Windows.  It should work now.

### CODE REVIEW

**Comment by keithrfung** [comment]
Those code scanning results are definitely off. This might need to be a tweak to code ql

File: pyproject.toml [hunk 1]
```diff
@@ -23,15 +23,15 @@ python-multipart = "^0.0.5"
 
 
 [tool.poetry.dev-dependencies]
-black = { version = "20.8b1", allow-prereleases = true }
+black = "^21.12b0"
```
Comment (keithrfung) [inline pyproject.toml:26]: This should match the `electionguard-python`. We have learned the hard way though not to use `^`. The version of black changes too often. 

https://github.com/microsoft/electionguard-python/blob/db1287d66c8aabf70a9d338bf6c249578906b3df/pyproject.toml#L43

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -51,8 +51,8 @@ jobs:
         run: make environment
       - name: Run Integration Tests
         run: make test-integration
-#      - name: Run Postman Tests
-#        run: make docker-postman-test
+  #      - name: Run Postman Tests
```
Comment (keithrfung) [inline .github/workflows/release.yml:54]: Honestly, we can just pull these lines.

**Review by keithrfung — CHANGES_REQUESTED** [review]
Couple minor changes. This on the surface looks good. 

In the future, would be good to separate out the pieces. I think there should have just been a upgrade python to 3.9 PR which this kind of hides this upgrade.

**Comment by keithrfung** [comment]
`make auto-lint` might fix those linting errors for that last one.

**Comment by SteveMaier-IRT** [comment]
> 
The change to python 3.9 was actually not planned just a result of the CI/CD build issues that were being seen.

**Review by SteveMaier-IRT — APPROVED** [review]

**Pull request merged**